code/modules/clothing/suits/bio.dm

```diff
@@ -35,9 +38,12 @@
 	burn_state = FIRE_PROOF
 
 	sprite_sheets = list(
+		"Tajaran" = 'icons/mob/species/tajaran/suit.dmi',
+		"Unathi" = 'icons/mob/species/unathi/suit.dmi',
+		"Vulpkanin" = 'icons/mob/species/vulpkanin/suit.dmi',
 		"Vox" = 'icons/mob/species/vox/suit.dmi'
 		)
-	hide_tail_by_species = list("Vox")
+	hide_tail_by_species = list("Vox, Unathi, Tajaran, Vulpkanin")

```

Comment:
Since this is all tailed species we might just want to give the suit the hide_tail flag